tests/arc/tcontrolflow.nim

```diff
@@ -1,12 +1,12 @@
 discard """
```

Comment:
Er, but this is a spec. The scope is ``{if cond: action}``.

Reply:
Yes, but `cond` is an escaping expression, and must be destroyed in the parents scope. Otherwise it might not be destroyed if the condition turns out to be false.

Reply:
Hmm ok.